ddl/partition.go

```diff
@@ -327,11 +327,6 @@ func buildTablePartitionInfo(ctx sessionctx.Context, s *ast.CreateTableStmt) (*m
 		}
 		pi.Expr = buf.String()
 	} else if s.Partition.ColumnNames != nil {
-		// TODO: Support multiple columns for 'PARTITION BY RANGE COLUMNS'.
```

Comment:
Why this is removed? we can't partition pruning for multiple columns, so this is not supported yet

Reply:
Oh, it's a duplicated check, the previous code already do this check.